lib/game.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class Game < Item
+  attr_accessor :publisher, :last_played_at
+
+  def initialize(publish_date = 'Undefined', publisher = 'Undefined', last_played_at = 'Undefined', archived: false)
+    super(publish_date, archived: archived)
+    @publisher = publisher
+    @last_played_at = last_played_at
+  end
+
+  private
+
+  def can_be_archieved?
+    super && @last_played_at > 2
+  end
```

Comment:
[Required] Kindly check the date of the last_played_at with a date conversation not as a regular integer.
- Instead you can use `Date.today.year - Date.parse(@last_played_at).year > 2` when comparing the last played year

Reply:
Oh, I get that. Thank you @darikmohammed.

Reply:
Done.